fenix/app/src/main/java/org/mozilla/fenix/trackingprotection/ProtectionsStore.kt

```diff
@@ -109,6 +129,57 @@ data class ProtectionsState(
     }
 }
 
+/**
+ * CookieBannerUIMode - contains a description and icon that will be shown
+ * in the protection view.
+ */
+@Parcelize
+enum class CookieBannerUIMode(
+    @StringRes val description: Int? = null,
+    @DrawableRes val icon: Int? = null,
+) : Parcelable {
+
+    /**
+     * ENABLE - The site domain wasn't added to the list of exceptions.
+     */
+    ENABLE(
+        R.string.reduce_cookie_banner_on_for_site,
+        R.drawable.ic_cookies_enabled,
+    ),
+
+    /**
+     * DISABLE - The site domain was added to the exceptions list.
+     * The cookie banner reducer will not work.
+     */
+    DISABLE(
+        R.string.reduce_cookie_banner_off_for_site,
+        R.drawable.ic_cookies_disabled,
+    ),
+
+    /**
+     * SITE_NOT_SUPPORTED -  The site domain isn't added to the exceptions list,
+     * but the cookie banner reducer didn't work.
```

Comment:
nit: Maybe we could re-word "but the cookie banner reducer didn't work"  to "because an unexpected error happened" ?

Reply:
Done